sdk/src/wallet/account/operations/syncing/transactions.rs

```diff
@@ -100,9 +100,9 @@ where
             if let Some(block_id) = transaction.block_id {
                 match self.client().get_block_metadata(&block_id).await {
                     Ok(metadata) => {
-                        if let Some(inclusion_state) = metadata.ledger_inclusion_state {
-                            match inclusion_state {
-                                LedgerInclusionState::Included => {
+                        if let Some(tx_state) = metadata.tx_state {
+                            match tx_state {
+                                TransactionState::Finalized | TransactionState::Confirmed => {
```

Comment:
Is it fine to do it for both ? I'm not yet fully sure when confirmed could be "reverted"

Reply:
I'm also not sure, but from what I understand we would also get the outputs from the indexer at that point
But `InclusionState` in the wallet currently has no finalized variant, so can't add this there
Fine if I just add a TODO?

Reply:
Fine by me

Reply:
But we should also try to get an answer from someone